setup/pom.xml

```diff
@@ -63,12 +63,6 @@
       <artifactId>jenkins-war</artifactId>
       <version>${jenkins.version}</version>
       <type>executable-war</type>
-      <exclusions>
-        <exclusion>
-          <groupId>org.jenkins-ci.modules</groupId>
-          <artifactId>sshd</artifactId>
-        </exclusion>
-      </exclusions>
```

Comment:
@oleg-nenashev I've added the excluded SSHD module back to jar. 
<img width="844" alt="image" src="https://user-images.githubusercontent.com/5915092/66488930-675eba00-eacc-11e9-81ad-1736e554a666.png">